index.html

```diff
@@ -112,6 +127,7 @@ <h1 class = "title">BOOK.com</h1>
                     <button  class="tag">javascript</button>
                 </div>
                 <button type="submit" class="normal-button">See Project</button>
+              </div>  
             </div>    
         </section> 
  <section id="about_me">
```

Comment:
- [ ] Your design needs some modification so it can be like the Figma one, kindly add a ```border-radius``` to the top right of the about section
- [ ] You need to add ```Frameworks``` and ```Skills``` to the right section after ```Languages```, please stick to the design as much as possible
- [ ] There is no space after the about section, kindly remove the space between the about and contact sections
- [ ] The ```Get My Resume``` button needs some more padding and color change when hovering over or clicking on it, please try to replicate the Figma design as much as possible
- [ ] The three languges cards are not the same, you must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1), please make the changes

- Your design:
![Screenshot_20220507_114757](https://user-images.githubusercontent.com/25227022/167253082-a65898d8-8f03-44e8-b635-0d5f19bf13e1.png)

- Figma design:
![Screenshot_20220507_114813](https://user-images.githubusercontent.com/25227022/167253104-28632096-2bb8-41b0-b44d-f1870b05d8d9.png)

Reply:
- [ ] @Ashe546 Great work implementing the changes, however there are few errors still to fix, kindly check the comparison below:


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
<img width="424" alt="image" src="https://user-images.githubusercontent.com/61408860/167266497-5fe1428c-fce8-4a65-bfb5-54355fd0d116.png"> |  <img width="515" alt="image" src="https://user-images.githubusercontent.com/61408860/167266529-9cc60d30-fc7d-442e-8776-13d8cc957b95.png">